### PR TITLE
Fixes #902

### LINKED ISSUE #902: LDA-Science.ipynb generate_griffiths_data() error
I was running the example and cannot proceed at this line:

`known_alpha, known_beta, documents, topic_mixtures = generate_griffiths_data(
    num_documents=num_documents, num_topics=10)`

Here is the output:

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-5-b8fb4c0de8c1> in <module>()
      2 num_documents = 6000
      3 known_alpha, known_beta, documents, topic_mixtures = generate_griffiths_data(
----> 4     num_documents=num_documents, num_topics=10)
      5 # num_topics, vocabulary_size = known_beta.shape
      6 

~/SageMaker/lda_topic_modeling_2019-10-11/generate_example_data.py in generate_griffiths_data(num_documents, average_document_length, num_topics, alpha, eta, seed)
    112             topic_index = np.argmax(word_topic)
    113             topic_word_distribution = beta[topic_index]
--> 114             word = sp.stats.multinomial.rvs(1, topic_word_distribution, size=1).reshape(vocabulary_size)
    115             documents[m] += word
    116 

~/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/scipy/stats/_multivariate.py in rvs(self, n, p, size, random_state)
   3218         n, p, npcond = self._process_parameters(n, p)
   3219         random_state = self._get_random_state(random_state)
-> 3220         return random_state.multinomial(n, p, size)
   3221 
   3222 

mtrand.pyx in numpy.random.mtrand.RandomState.multinomial()

common.pyx in numpy.random.common.check_array_constraint()

ValueError: pvals < 0, pvals > 1 or pvals contains NaNs

### PR DESCRIPTION
**Pull request by hsl89**
*Issue #, if available:*
A fix for #902 

*Description of changes:*
Probabilities used for multinomial distribution sampled from Dirichlet dist sometimes exceed 1 for numerical precision reasons. I normalized the probability so that the sum of prob is slightly less than 1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aaronmarkham — CHANGES_REQUESTED** [review]
Please clear outputs in your notebook before adding it to your commit. (This will reduce the changes a lot.)
It seems like if you do this, there are no changes to the notebook... just to the .py file. Is that right?

File: introduction_to_amazon_algorithms/lda_topic_modeling/LDA-Introduction.ipynb [hunk 1]
```diff
@@ -117,15 +118,34 @@
...
      "parameters"
     ]
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Couldn't call 'get_role' to get Role ARN from role name hongshan to get Role path.\n"
```
Comment (aaronmarkham) [inline introduction_to_amazon_algorithms/lda_topic_modeling/LDA-Introduction.ipynb:133]: Definitely don't want this kind of thing in there...

Comment (hsl89) [reply introduction_to_amazon_algorithms/lda_topic_modeling/LDA-Introduction.ipynb]: cleared in this commit b59d325

**Review by aaronmarkham — APPROVED** [review]
LGTM

**Review by ChuyangDeng — APPROVED** [review]

**Pull request merged**